### ISSUE TITLE
SickRage

### ISSUE THREAD

**Issue by OmgImAlexis** [SickRage/old-sickrage-issues#3]
We've moved repos, this is the new one. Everything will be explained in a news post within the next week.
We're also not providing support in the old repo so please open an issue in this repo if you need help.

To change over to this repo shutdown SickRage and then run these commands in the SickRage directory.

```
git remote set-url origin https://github.com/SickRage/SickRage.git
git fetch origin
git reset --hard origin/master
```

After this you should be able to start SickRage.

If you're using Arch Linux there's a package on [AUR](https://aur.archlinux.org/packages/sickrage2-git/), keep in mind this was uploaded by a user and not the developers so we hold no responsability if it doesn't work. We always suggest you use git to install SickRage.

Short info: http://pastebin.com/raw.php?i=XgMreCb2
Mirror of everything that's been going on: https://wvvw.me/mirror/sickrage/

**Comment by OmgImAlexis** [comment]
@Amadeus-

**Comment by miigotu** [comment]
@adaur

**Comment by adaur** [comment]
Why did you want to loose all the stars?

**Comment by OmgImAlexis** [comment]
@adaur huh?

**Comment by OmgImAlexis** [comment]
@sjay1

**Comment by OmgImAlexis** [comment]
@XxUnkn0wnxX

**Comment by CristianBB** [comment]
Thanks for your notice @OmgImAlexis , can I migrate my PR to this repo? To develop branch of course

**Comment by miigotu** [comment]
Yes, but I still think it needs a few changes. Shifting a few things will make a lot of unneeded code go away. We can talk about it after you PR here =P

**Comment by fernandog** [comment]
@ncksol

**Comment by ncksol** [comment]
thnx for the ping @fernandog

**Comment by adaur** [comment]
@MGaetan89

**Comment by MGaetan89** [comment]
Thanks for the ping.
Was away from SR a while and missed quite a few things it seems.

**Comment by fernandog** [comment]
@Arcanemagus

**Comment by Arcanemagus** [comment]
Oooo, thanks for the ping, hadn't had a chance to hunt this fork down yet :stuck_out_tongue:.

**Comment by phaseburn** [comment]
Thanks for the ping. Seeing as how the old channel is semi-defunct, is there a new IRC channel?

**Comment by ishkanan** [comment]
Thanks for the ping. Will keep an eye open for the news post.

**Comment by adaur** [comment]
#sickrage-issues

**Comment by fernandog** [comment]
@disrani

**Comment by disrani** [comment]
So patches should be submitted here rather than SICKRAGETV?

**Comment by duramato** [comment]
yes they should

**Comment by duramato** [comment]
@Hellowlol

**Comment by miigotu** [comment]
tempora

**Comment by DisasteR** [comment]
So all members have joined here ?

**Comment by miigotu** [comment]
Yes, screenshot proof here: http://goo.gl/gWWiIc

@SickRage/collaborators @SickRage/moderators should we make the teams public now?

**Comment by OmgImAlexis** [comment]
Yes!

**Comment by DisasteR** [comment]
What about scène exception and time zone repository ?

**Comment by OmgImAlexis** [comment]
@DisasteR They've all been moved over to this repo.

**Comment by DisasteR** [comment]
I think put the team list  public will help people.

**Comment by DisasteR** [comment]
@OmgImAlexis  not seing them on group page...

**Comment by miigotu** [comment]
The org layout is a bit different so we dont have 7 repos
https://github.com/SickRage/sickrage.github.io

The url paths are still the same as before, but this way we get higher api limits and cleaner organizing.

**Comment by OmgImAlexis** [comment]
@johndooe

**Comment by DisasteR** [comment]
@miigotu  Thanks for clarify.

**Comment by alisonatwork** [comment]
:+1: for making the team public, looking forward to continuing development and contributions over here.

**Comment by miigotu** [comment]
Yeah, as long as no one has a problem with it public, it is better for users to see that it isnt just some random takeover/fork

**Comment by Rouzax** [comment]
@miigotu thanks for the clarification.
Maybe this was a great time to change the name :smiley:

**Comment by miigotu** [comment]
@Rouzax we hope to in the near future, but not because of the move. SickRage name just doesn't make sense anymore with the downing of tvrage. We hope to totally rebrand eventually to a name that is totally non-reliant on the existence of an external service and is more descriptive of what we offer.

For the time being though, keeping the name during user transition is important, so they can make the connection that this is a migration of the project and team and is still the same people who they have trusted for over a year.

Last thing we want is for our loyal and potential users to suffer or miss out.

Once the word is out and the user-base has migrated and comfortable, the plan that has been talked about so far is that we will take suggestions on the new name and then the team will select some from the pool of suggestions and then let users vote on the final result. This process might change, but it seems like a decent approach.

**Comment by disrani** [comment]
FYI, the News page still has links to the old github repos. I couldn't figure out how that page gets generated.

**Comment by miigotu** [comment]
@disrani will look at that now and fix it

**Comment by miigotu** [comment]
Done

**Comment by Rouzax** [comment]
@miigotu: That makes sense.

BTW after doing:
```
git remote set-url origin https://github.com/SickRage/SickRage.git
git fetch origin
git reset --hard origin/master
```
One other thing I noticed was that the old repo was still in my config.ini so I opted to make a new clone and only copy the settings I needed back into the config.ini

**Comment by miigotu** [comment]
@Rouzax we have a commit that auto-updates that git_remote_url for you =P

**Comment by Rouzax** [comment]
@miigotu And I just did it manually :laughing:

**Comment by Smultie** [comment]
Thanks for the news. Updated without a problem.

**Comment by nbara** [comment]
Hi, I get an error when I run the above commands: 

"10:25:51 WARNING::MAIN :: Unable to setup GitHub properly. GitHub will not be available. Error: 
10:25:51 INFO::TORNADO :: Starting SickRage on http://0.0.0.0:8081/ "

Do you have an idea what went wrong?

**Comment by adaur** [comment]
Did you stop sickrage before the git commands?

**Comment by nbara** [comment]
Yes, I did.

**Comment by Rouzax** [comment]
@nbara 
Just save these files somewhere else:
* cache.db
* config.ini
* failed.db
* sickbeard.db

After that remove SickRage and create a new clone
```
git clone https://github.com/SickRage/SickRage.git c:\SickRage
```
and copy the files back and start SickRage.

**Comment by nbara** [comment]
Mmh... I still get the error I'm afraid.

**Comment by labrys** [comment]
What OS?  If it's Windows, make sure the Git directory is in your PATH.

**Comment by miigotu** [comment]
@nbara

Rename your old SR install.
Then use this:
https://github.com/VinceVal/SickRageInstaller/releases/latest <-- this will install SR from the new org/repo

Then copy config.ini and sickbeard.db from the old source into the install, to the same location where it was on the other source.

Start SR and make sure your data is there.
Delete old source.

**Comment by nbara** [comment]
I'm on OSX 10.11. 

I managed to get it to work with @Rouzax 's solution above. However I also had to manually edit my former config.ini file for some reason.

These lines in particular were causing problems:
`git_autoissues = 0`
`git_username = "" `
`git_password = ""  `

(auto issues was set to 1, and username and password were filled).

Anyway, all good now. Thanks for the help, and good luck with the new start. :)

**Comment by adaur** [comment]
@raver2046

**Comment by fernandog** [comment]
@Noktam

**Comment by Arcanemagus** [comment]
@nbara I had the same issue, turns out my GitHub credentials were wrong and I just hadn't noticed until checking after switching things. If you have 2 factor auth enabled you will need to create a token (with no privileges) and put that as the "user name" for it to work.

Funny story, I had sent echel0n a PM on the old forums asking him what proof there would be that he wouldn't just abandon it again after kicking everyone out... and now I'm permanently banned :stuck_out_tongue:.

**Comment by Hellowlol** [comment]
I don't think that was a funny story. It's stupid question and it got you a stupid response :D

**Comment by Fetchered** [comment]
Question about migrating to the new SickRage on a QNAP NAS. When attempting to run the first command (git remote set-url origin https://github.com/SickRage/SickRage.git), I receive a fatal error:

/share/CACHEDEV1_DATA/.qpkg/QSickRage] # git remote set-url origin https://github.com/SickRage/SickRage.git
fatal: Not a git repository (or any parent up to mount point /share/CACHEDEV1_DATA)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).

Same problem running this from the next sub-folder SickRage. Manually edit the config.ini with the new URL and git fetch origin gives me the same error.

Any thoughts on a way to do this cleanly on a QNAP NAS?

**Comment by Arcanemagus** [comment]
@Fetchered you might want to check out https://github.com/SickRage/sickrage-issues/issues/6, another user already worked through it on your platform.

**Comment by Fetchered** [comment]
@Arcanemagus Thanks for the help. It didn't relate to the issue I had, but I ended up having his issue after I resolved mine. I originally installed via qpkg which is why the git command wasn't working. My process to manually complete on QNAP was as follows (hope this helps someone):

Copy the recommended files as mentioned by @Rouzax (cache.db/config.ini/failed.db/sickbeard.db) from the SickRage/DATAS folder, but also copy the /share/CACHEDEV1_DATA/.qpkg/QSickRage/SickRage/CONFIG/sickbeard.conf file. 

Run command './QSickRage.sh stop'
Rename QSickRage/SickRage to QSickRage/old_SickRage
from QSickRage/ run 'git clone https://github.com/SickRage/SickRage.git'
Once clone is complete, create folders SickRage/CONFIG and SickRage/DATAS
put backed up files in their respective locations
run the following from the QSickRage/SickRage folder (as detailed by @OmgImAlexis ):

git remote set-url origin https://github.com/SickRage/SickRage.git
git fetch origin
git reset --hard origin/master

modify the sickbeard.conf file if required (mine needed to be changed, to match the config.ini)
Then I had to follow the solution at #6 (sslverify=false) 

Once complete, run './QSickRage.sh start' from the QSickRage folder, and SickRage should start properly.

Again, hope this helps someone, and thanks again for your help.

**Comment by gosku** [comment]
Thank You for the clarification guys. Changing Origin. I wish all the problems had this easy solution :)

**Comment by OmgImAlexis** [comment]
@Hydrog3n @harmakhis971

**Comment by memmons-ntst** [comment]
:+1: On the transparency of what's going on.

**Comment by maxcore** [comment]
Thank you all staff for working everyday every night on sickrage!

**Comment by OmgImAlexis** [comment]
@memmons-ntst that's one thing we wanted to get right this time. We want users to know exactly what's going on. Then they can decide what they want to do. :smiley:

**Comment by Hydrog3n** [comment]
@OmgImAlexis Why you mention me ? I know the fork ;) http://mondedie.fr/viewtopic.php?pid=30327#p30327 

thank to @adaur

**Comment by miigotu** [comment]
@Hydrog3n just attempting to inform as many people as possible, so they can decide if they want to contribute here or use the new org.

**Comment by Hydrog3n** [comment]
Ok Ok ! 

Few day I was wondering why my sickrage doesn't have update. Now it's ok :+1: 

Good luck :four_leaf_clover:

**Comment by seed1234** [comment]
hey guys 

how would you run these commands on freenas 
I mean in the sickrage jail

**Comment by veribaka** [comment]
I assume this output is norma:

HEAD is now at ea24e7b Merge pull request #96 from SickRage/title_error

**Comment by OmgImAlexis** [comment]
@veribaka yes. That's correct.

**Comment by labrys** [comment]
Lol we need to push a new commit to master just to change the name... thats the third time ive seen that mentioned

**Comment by Mul-tiMedia** [comment]
Just switched to the new repository! Keep up the good work!

**Comment by XxUnkn0wnxX** [comment]
I'm so confused which one is the real sickrage???? as i been having issues with mine and thinking of doing a reset but unsure which one....

**Comment by Rouzax** [comment]
@XxUnkn0wnxX Please read: http://pastebin.com/raw.php?i=XgMreCb2 and decide for your own.
This SickRage is a spinoff of the "original" because of internal differences between the members, all of the dev's that have been developing SIckRage for the past year have started this branch

**Comment by duramato** [comment]
@Rouzax This isn't really a spinoff as all of the team is here

**Comment by Rouzax** [comment]
@duramato Didn't really know how to objectively describe it otherwise :laughing:

**Comment by adaur** [comment]
More like a fork of a fork

**Comment by miigotu** [comment]
More like the fork that was SiCKRAGETV just moved to SickRage. The only difference at all is the org change and a booted echelon.

**Comment by labrys** [comment]
It's a spooooon damnit! :grinning: ![SPOON] (http://static.comicvine.com/uploads/original/7/79709/1892621-the_tick_spoon_by_tigerofmyth.jpg)

**Comment by miigotu** [comment]
Thats not a spoon, its a tick! lol

**Comment by nbara** [comment]
Is there an ongoing thread/forum somewhere where we could follow ongoing discussions about the recent drama?

**Comment by labrys** [comment]
Ongoing, no, but there are pasties of the drama available.  I'll post a link when I get home if nobody beats me to it... Personaly I'm just ready to move on...

**Comment by veribaka** [comment]
nbara, I recommend you read this 
https://wvvw.me/mirror/sickrage/

The pasties labrys mentioned.

**Comment by nbara** [comment]
Yes I've seen those. And TBH I don't find them as compelling as you would like us to believe (?).

As a simple SR user, I've been very distant to all of this mess, and the only reason I switched to this repo is because most of the active devs did. However there is no denying that echel0n has also given a lot of his time to SR, and I can't help but feel bad about the whole situation. 

Hence, I was asking whether there were any new developments since the IRC discussions you are posting everywhere.

Don't get me wrong, I'm very grateful to all of you for making SR better, I just wish we had more information about the whole situation as it unfolds.

**Comment by miigotu** [comment]
There is no more information. We are not dwelling in the past. We also are not focusing on trying to prove ech3l0n is a bad guy, that is for the birds.

Bottom line the project has moved, echelon will not be a part of it anymore. He spent a few months working on it and disappeared for over a year. The project has outgrown him and he has caused the project to almost completely die for the second time. we refuse to let that happen to our users again.

**Comment by veribaka** [comment]
I'm exactly in the same boat as you nbara. I only know what little information there is in that pastebin.

**Comment by labrys** [comment]
I originally had planned to contribute to both, but I don't care for the response I received from echel0n so it made my decision easy.  Really the drama that caused it is irrelevant.  If you want to move on with all the SickRage dev team, we are here.  If you prefer to stick with echel0n's repository, you know where that is too.  The team here has been highly active and dedicated and I see it staying that way.  Echel0n has not been as active as of late.  Take your pick.  Switching between the two is fairly painless so its nothing that needs to be agonized over.

**Comment by neoatomic** [comment]
As an enduser the only thing relevant to know is that the development and support are continuing here, and that you can use/enjoy Sickrage as before. :)
Just leave the drama for what it is...

**Comment by labrys** [comment]
Well said @neoatomic

**Comment by OmgImAlexis** [comment]
To be completely honest this drama is almost good for the team, this is the most active we've been in months. :stuck_out_tongue:

**Comment by veribaka** [comment]
Good, now get to work. :P

**Comment by RabidWolf** [comment]
Will the Anime section be maintained or are you looking to get rid of it as I know miigotu has said in the past Anime is too much of a hassle.

**Comment by medariox** [comment]
@RabidWolf 
Just because someone of the team said it is a hassle (it is), it doesn't mean we are not going to support it. A person can still express personal opinions, don't you think? We are going to support all features that SickRage has currently and will be adding even more in the near future (actually we constantly are already). No one here is going to decide anything on his own regarding the future development of SickRage, in contrary to other places you most probably know about. All decisions will be made public and decided by the team with the help and feedback of the users.

**Comment by neoatomic** [comment]
@RabidWolf Dont worry, Anime wont get removed and is still maintained. :)

**Comment by RabidWolf** [comment]
Good to know, just wanted to know before moving to this version. :-)

**Comment by OmgImAlexis** [comment]
If anyone needs logs from issues they had opened looks like the old issue tracker is open. I suggest you get anything you need before it's closed again.
https://github.com/SiCKRAGETV/sickrage-issues/issues

**Comment by Nikola-K** [comment]
Well, the "switch" completely broke sickrage for me...

![image](https://cloud.githubusercontent.com/assets/720441/11279934/ae3cd08a-8ef3-11e5-8990-355012155071.png)

**Comment by duramato** [comment]
@Nikola-K clear your mako cache SickRage/cache/mako

**Comment by miigotu** [comment]
More like just restart SickRage, thats a one time error because the python that loads the page was updated, and hasnt been reloaded yet (but the mako template has which ises the new code). You dont need to clear the cache or anything.

**Comment by Nikola-K** [comment]
I did actually try to restart sickrage first @miigotu but still got the same error, clearing cache did the trick. Thanks @duramato !

**Comment by miigotu** [comment]
Ok, glad it works for you regardless =P

**Comment by deusxanime** [comment]
Looking forward to a revitalized SickRage (or whatever you guys decent to call it)!

**Comment by rEes9P** [comment]
Looking forward to switching as well. I am on Mac OS X 10.11 and am confused about what I am supposed to do with my .db files as one of the pages highly recommends starting out with no database files but there are no definite instructions on how to do this. Am I supposed to copy cache.db, config.ini, failed.db and sickbeard.db out of the folder and then copy them back after I run the following commands?

git remote set-url origin https://github.com/SickRage/SickRage.git
git fetch origin
git reset --hard origin/master

May I request that you make include easy to follow step-by-step instructions on how to migrate onto your version and have it on the main SickRage / SickRage GitHub page as well as in the pastie that you have linked to? Almost all the information is already there but it's not very clear.

**Comment by Arcanemagus** [comment]
I believe the section are are talking about is meant to be read by people coming from SickBeard. As you are simply coming from an older version of SickBeard you should be safe to just run the commands above. If you don't have backups elsewhere it couldn't hurt to keep a backup though.

**Comment by OmgImAlexis** [comment]
@dbrain @Frankster-NL if you want help from the devs you'll only get it here.

**Comment by OmgImAlexis** [comment]
@EternallyBored

**Comment by labrys** [comment]
@rEes9P You should not need to delete your database and can likely do an inplace update to this repository.  If you have issues the cache may need to be cleared but it should be fairly painless.

**Comment by miigotu** [comment]
@rEes9P in SickRage itself I would do a normal backup on ip:port/config/backuprestore/
Then do the commands above and you should be done.
The backup is just something you ALWAYS do before messing with your install, but this changeover is totally safe.

You might get a mako error until SR is restarted (and might need to clear cache/mako or not), but other than that it is a pretty smooth transition.

**Comment by mdklapwijk** [comment]
Thanks for the info Alexis, missed that. Found a better markedup version of the info probably from Miigotu:
- http://pastie.org/private/m7lryrqqihyeyl1supuepq

**Comment by OmgImAlexis** [comment]
@mdklapwijk that's actually where mine was mirrored from but we've already seen one of the screenshots disappear so I made sure to back it up on my server.

**Comment by skeeve42** [comment]
@OmgImAlexis Thanks for the ping and info!

**Comment by OmgImAlexis** [comment]
@xnick

**Comment by FlyGuy94** [comment]
Found this on the news site.

!!! IMPORTANT MEMBER ALERT !!!

IT HAS BEEN BROUGHT TO MY ATTENTION FORMER EX-TEAM MEMBERS OF SICKRAGE ARE SOLICIATING USERS TO SWITCH TO A NEW GITHUB REPO ADDRESS THAT IS IN NO WAY OFFICIAL OR OWNED BY ME, DOING SO COULD ALLOW THEM TO INFECT YOUR COMPUTER SYSTEMS WITH MALICAIOUS CODE THAT COULD WIPE YOUR ENTIRE SYSTEM OUT DATA AND ALL SO PLEASE REPORT ANY FURTHER ABUSE TO SICKRAGE.TV@GMAIL.COM

**Comment by adaur** [comment]
Yeah, it's not 'new' anymore actually :p 

We're malicaious and proud of it.

**Comment by FlyGuy94** [comment]
Ok i'm confused so there is the old sickrage the new one and sickgear. And they are all different things? So how do I change on a synology?

**Comment by adaur** [comment]
SickRage gave birth to both SickGear and the new SickRage.

We're the team that has been behind 'old' SR for over a year, until the owner of the repo kicked us out, so here we are. Have you read http://pastebin.com/raw.php?i=XgMreCb2 ?

**Comment by FlyGuy94** [comment]
I read it now. Sad stuff. Can't use the commands above on synology git is not installed?

**Comment by neoatomic** [comment]
@FlyGuy94 Here : https://github.com/SickRage/sickrage-issues/issues/14#issuecomment-158025087 

Or if you prefer SSH/GIT you can do  : 

```
* Make a backup for safety.
* Shutdown Sickrage in the package center.
* Use Putty and go to the following folder. :
/volume1/@appstore/sickrage/var/SickRage

* Give the following commands:
/volume1/@appstore/git/bin/git remote set-url origin https://github.com/SickRage/SickRage.git
/volume1/@appstore/git/bin/git fetch origin
/volume1/@appstore/git/bin/git reset --hard origin/master

* Start SickRage again in the package center.
```
I'm also working on a Wiki for the Synology. Hopefully i have that ready later today.

**Comment by FlyGuy94** [comment]
Thanks got it working not sure why you put the git commands in there they don't work on synology. Cheers

**Comment by willemw12** [comment]
@neoatomic: Should it not be:
- "Shutdown Sickrage"
- "Make a backup for safety"

Otherwise you backup an online database?

**Comment by neoatomic** [comment]
@willemw12 No, you make the backup inside Sickrage. If you Shutdown Sickrage first, you cant login/access it. Think you are referring to a manual backup witch WinSCP over SSH. Thats also a possibility but for most novice users not the easiest. :)

**Comment by neoatomic** [comment]
As promised i created a Wiki page for our Synology users.

https://github.com/SickRage/sickrage-issues/wiki/Switching-your-Synology's-Sickrage-to-the-new-repository

**Comment by channouze** [comment]
Thanks for the heads-up!

**Comment by duramato** [comment]
@ikriz

**Comment by ikriz** [comment]
@duramato Was already reading this ;)

**Comment by duramato** [comment]
@ikriz ahh ok ,just thought in letting you know :)

**Comment by ikriz** [comment]
i'm just kinda stunned this is even possible these days.. opensource projects still have to deal with humans I guess.

**Comment by duramato** [comment]
@loonielongbottom here`s the explanation

**Comment by VikingSven82** [comment]
I've followed the steps in the pastebin, all seems to have worked - is there a way to verify that I'm now running the 'new' SickRage branch?

**Comment by duramato** [comment]
@VikingSven82 check the help and info page and check the links

**Comment by alisonatwork** [comment]
It's not very clear, @VikingSven82, I agree. I had the same problem earlier when my branch switched back to the old repo due to the Windows install storing its config.ini elsewhere and git auto-updating. The only way I can see to tell via the GUI is to visit http://localhost:8081/news/ and check if there is a rant or a message about changing repo. If it's the former, the update may not have stuck.

**Comment by Dr-Bean** [comment]
@neoatomic Two suggestions for your instructions in https://github.com/SickRage/sickrage-issues/wiki/Switching-your-Synology's-Sickrage-to-the-new-repository#switching-with-sshgit:
- `official GIT package from Synology`. << There's no such package :) Synology provides a Git-Server package, and while it does have a `git` binary, it's located in a different place.
The SynoCommunity `SickRage` package already has a dependency on the SynoCommunity `git` package, and by default, that _is_ located in `/volume1/@appstore/git/bin/git`.
- Which brings me to the second point: people can choose to install packages on other volumes (e.g. `/volume2`). Those who have done that won't be able to c/p those commands.
Referring to `usr/local/git/bin/git` will solve that, because it points to whatever volume the package is installed to (via a symlink created during package installation).

**Comment by labrys** [comment]
@VikingSven82 you can check the News and the Changelog.. both are updated on this repo

**Comment by neoatomic** [comment]
@Dr-Bean Thanks for the clarification. Appreciate that.
I'm considering to remove that manual GIT part all together, reinstalling the package from the package center is the simplest and the safest for Syno users.

**Comment by dbrain** [comment]
@OmgImAlexis Thanks for the heads up to this repo. Fixed the issue I had in the old repo (post processing not working). Bit iffy about what's going on with SickRage at the moment, but your story sounds legit.

**Comment by OmgImAlexis** [comment]
For transparency sake here's the whole team. These are all of the devs now part of the @SickRage/collaborators team. Some of them haven't been on and changed to public yet. Once they do you'll be able to see them all on the team page [here](https://github.com/orgs/SickRage/people).

![screen shot 2015-11-22 at 1 53 00 pm](https://cloud.githubusercontent.com/assets/6525926/11321885/60bcccaa-9120-11e5-8f8b-ea5331c6d0a7.png)

**Comment by OmgImAlexis** [comment]
@bacevs

**Comment by OmgImAlexis** [comment]
@DonZalmrol

**Comment by OmgImAlexis** [comment]
@brando56894, you may want to open the issue on this repo so it actually gets looked into.

@chonkworld

**Comment by chonkworld** [comment]
@OmgImAlexis Thanks for the heads up!

**Comment by OmgImAlexis** [comment]
@benyjr, if you've got 2 step enabled you'll need to setup a Personal Access Token and use that as your password. https://github.com/settings/tokens

**Comment by benyjr** [comment]
@OmgImAlexis That's weird, because I double checked and I don't have two-factor authentication turned on.
![2015-11-24 13_58_53-security](https://cloud.githubusercontent.com/assets/14084052/11377424/a71a02a0-92b3-11e5-8e5b-b1707d05bab5.png)

**Comment by OmgImAlexis** [comment]
@adrienkohlbecker

**Comment by OmgImAlexis** [comment]
@benyjr that's odd. Jump on IRC and I'm sure someone will be able to help you out.

**Comment by Zipties** [comment]
I am really torn on if I should switch repos or not. The stability of the project seemed to go pretty downhill the last few months, but I doubt echel0n is going ot keep the old one going all by himself.

**Comment by labrys** [comment]
@Zipties that's actually our primary goal right now.  We are working on updating everything for PEP, and lint, re-evaluating logic flow, and revamping our unit testing to cover more cases with the eventual goal of having an exhaustive and robust test suite.

**Comment by Zipties** [comment]
@labrys That is good to hear. I hope the team is a little less trigger happy moving forward to committing changes. I am noticing that the post processor seems to be working for me again on the current build. it had been freezing up and killing the show searcher for me for weeks.

**Comment by duramato** [comment]
@Zipties those problems are all related with the team being kicked and they are only existing on the old repo ,also they where all fixed in a matter of hours here...

**Comment by Zipties** [comment]
The issues I was referring to have existed for months.

**Comment by duramato** [comment]
@Zipties Example? just curious ... :P

**Comment by alisonatwork** [comment]
This is quite a tangent, @Zipties, can you open a new issue? To be honest I have had the preprocessor die several times over the past weeks for me too, but it happens inconsistently so I haven't dug deeper. I am wondering if it is just a Windows thing, too. I don't think there are any serious Windows developers of SickRage (I only switched about a month ago when the installer became available).

**Comment by labrys** [comment]
@alisonatwork actually I do all my dev work and testing on Windows.  XP, Vista, 7, 8.1 and 10 primarily.

**Comment by Zipties** [comment]
Not a tangent, just stating how unstable things got around the mako change over for me on windows. I can re open tickets I had opened for the old guy but I was just told it was related to tracker changes and would be resolved soon.

The biggest issue was post processor freezing, because when it did it spiked cpu usage and caused show searching to stop. Only fix was to delete the cache. This went for for weeks until I disabled scanning and used scripts, but they had serious issues too. Like nzbtomedia doesn't seem to be able to hardlink on windows.

Game of thrones season 5 episode 10 downloads every few days even though it is correctly set as already being downloaded at that quality.

I have over 800 warnings right now from shows that won't process because an existing file is present at a larger size. This issue is a couple of weeks old.

Again not a tangent. I am donator and you guys do great work. I simply noticed SR went from a set it and leave it solution to one that needed bi daily maintance.

**Comment by labrys** [comment]
Actually @Zipties if you wouldn't mind opening an issue and posting your logs it would help with some of the updates to the testing I'm working on and some ideas I have for error logging.

**Comment by Zipties** [comment]
@labrys Sure, which issue are you interested in tracking down? The post processor hasn't frozen yet so I don't have a current log for that. It may very well be fixed now as it is working far better than it has in a while. I have over 3,000 warnings for files it wont process because the current files are larger. It seems to stack on each scan.

**Comment by VinceVal** [comment]
@Zipties with the issue of "current files are larger," is that not really the case? Do any current files exist, and if so, I assume they're smaller? If so, that seems like a legitimate issue and we'd appreciate if you could open an issue about that so we can figure out what's going on. Thanks!

**Comment by OmgImAlexis** [comment]
@felipesalomao if you'd like help please use this repo and the issue tracker here and not the old one.

**Comment by felipesalomao** [comment]
Ok, i will.. Thank you

**Comment by fernandog** [comment]
join irc sickrage-issues

**Comment by Spippo** [comment]
Anyone knows how to switch to this repo in FreeNAS?
As there is no git

**Comment by duramato** [comment]
@doczong

**Comment by doczong** [comment]
@duramato... yeaaah, sorry, this is a fork.
Don't get me wrong, I like what you guys are doing, but lets also not try and rewrite history, this is not George Orwell's 1984.

**Comment by duramato** [comment]
Call what you want but all the of team is here and active developing is happening here soo..

**Comment by miigotu** [comment]
Its just another chip from the same bag dude. You are free to choose for yourself, AND so is everyone else. We have been nothing but an open book during this whole thing, and we don't tell users lies or spread rumors or block the whole world who has an opposing view. Trust me, there is a lot more to the story that we don't tell because it would be unprofessional. Like, aren't telling users that an open source project is going to infect their computers, for instance.

Users cant even get support in that other repo, let alone any decent updates.

Stop acting like this is a personal thing, because we did what was right for our users. Thats all there is to it.

**Comment by miigotu** [comment]
@Spippo Im not very familiar with freenas, but you can install git on freenas for sure. The FreeNas package is supposed to be update soon if it is not already.

**Comment by labrys** [comment]
To my knowledge the FreeNAS update is still pending.

**Comment by adaur** [comment]
@justinmg

**Comment by adaur** [comment]
@justinsg

**Comment by Kawalskint** [comment]
BRANCH: (master) / COMMIT: (d294e9faf48d45051a50d4e2cf3a0252c28d5b17) 
OS: Windows 8.1
Python Version: 2.7.9

How do I change repo on Windows without losing my entire DB? Afaik I don't have an option "To change over to this repo shutdown SickRage and then run these commands in the SickRage directory." on windows OS. 

As I am on the old repo I currently have 88 errors listed:

SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents] :: Failed parsing provider. Traceback: 'Traceback (most recent call last):\n  File "C:\\SickRage\\sickbeard\\providers\\kat.py", line 102, in _doSearch\n    data = xmltodict.parse(HTMLParser.HTMLParser().unescape(data.encode(\'utf-8\')).decode(\'utf-8\').replace(\'&\', \'&\'))\n  File "C:\\Python27\\lib\\encodings\\cp1252.py", line 15, in decode\n    return codecs.charmap_decode(input,errors,decoding_table)\nUnicodeDecodeError: \'charmap\' codec can\'t decode byte 0x8f in position 10764: character maps to \n'

**Comment by Rouzax** [comment]
@Kawalskint you could create a backup in the "Old" SickRageTV and after that rename the folder. Install the new SickRage and restore the previously made backup.

**Comment by labrys** [comment]
@Kawalskint you can also do an update via command line and if you login to IRC I can talk you through it.  [Kiwiirc](kiwiirc.com) is a web IRC client and just `/join #sickrage-issues`

**Comment by Kawalskint** [comment]
@labrys thanks, I get Error Connecting (ETIMEDOUT) from https://kiwiirc.com/ and "This page can’t be displayed" from https://webchat.freenode.net, maybe down right now?

**Comment by labrys** [comment]
@Kawalskint no probs glad we could get you sorted :grinning:

**Comment by labrys** [comment]
@felipesalomao [SickRage](https://github.com/SickRage/SickRage) has moved here.  We have made quite a few changes to subtitles.  If you would like any assistance switching to the new repository please let us know.  I don't believe @medariox or @fernandog is still watching the old sickrage-issues.  If you have any further issues with subtitles after the switch you can post them in our issue tracker here at [SickRage/sickrage-issues](https://github.com/SickRage/sickrage-issues).

**Comment by labrys** [comment]
@bigmattp @polyneux @ProjectLane @mvarrieur  The [SickRage](https://github.com/SickRage/SickRage) development team has moved here.  If you would like any assistance switching to the new repository please let us know.  If you have any further issues with providers after the switch you can post them in our issue tracker here at [SickRage/sickrage-issues](https://github.com/SickRage/sickrage-issues).

**Comment by labrys** [comment]
The [SickRage](https://github.com/SickRage/SickRage) development team has moved here.  If you would like any assistance switching to the new repository please let us know.  If you have any further issues with after the switch you can post them in our issue tracker here at [SickRage/sickrage-issues](https://github.com/SickRage/sickrage-issues).

@aiiDaH @CYR4N0 We have made a lot of changes to post-processing and more are on the way.
@didyouexpectthat @Nico31300 @squareflapjack We have also made a lot of changes to subtitles.
@Kamolak You can make feature requests on [Feathub](http://feathub.com/SickRage/SickRage)
@victorclaessen Please let us know if you have any tornado errors after the switch.
@rati0nal Let us know if you have any issues with transmission after the switch.

**Comment by labrys** [comment]
The [SickRage](https://github.com/SickRage/SickRage) development team has moved here.  If you would like any assistance switching to the new repository please let us know.  If you have any further issues after the switch you can post them in our issue tracker here at [SickRage/sickrage-issues](https://github.com/SickRage/sickrage-issues).

@huush Your error with `NoneType` objects has been fixed here in develop.
@rebelmethod If you continue to have issues with searching HDSpace after switching please let us know @roccoss If you would like assistance with your provider issues, please open an issue request on our tracker here after switching, and we'll see what we can do for you.
@juju35 Please let us know if you have any tornado errors after the switch.

**Comment by Yellow1144** [comment]
Hi..
Well, looks like I'm following you here now. I tried updating my 'other' version last night and it would crash on launch. Tried to do some debugging but there were so many problems, I just abandoned ship. I bet it'll die, he'll get tired and pack it in in a while. Given that..

I tried to manually install your zip, it would never start. Once I just used the files in your zip to update my current one, It worked fine, but it could not see updates, despite everything pointing to this new Git. . I read this thread and decided to try the installer.. 

Hmm. I am a bit miffed that the installer could not see that i already have Python 2.7.10 and the latest Git already on my system. I also already had it running as a service too. So now I have two Python and Git installs. I understand the desire to have an installer for those who are totally new but for those of us who are just moving over from the other version or SickBeard, either improve the installer to detect existing installations and make no changes to those, or provide a version that is complete on its own and we can manually install. 

It also took me a while to find that the config and databases from my previous install needed to be placed in the new Data folder instead of where they used to be. For those who are porting from a previous install this is an important note.

It'll be interesting to see where you guys take it.

**Comment by miigotu** [comment]
@Yellow1144 glad for you to join us. Anyways, the python and git that come with the installer are just portable versions. We saw it as a "best in all cases" approach to have a little control over what versions and libraries are installed in it. It will not effect your system wide versions in any way, and other apps wont be affected by it.

Seems you just made 2 different switches at once which made this a bit more confusing for you, switching to the new repo and also switching the way SR was installed on your system =P.

Ofc if we can make it easier in the future that is always the goal.

**Comment by TagForce** [comment]
I won't be asking you wth happened (I've read, seen, and drawn my conclusions)... So just to let you guys know I've switched over and look forward to finally picking up where I left off a couple months ago.

Need to read up on the changes in the code while I was away, so give me some time. I can safely say it's good to be back, and can't wait to improve this already great product even further.

**Comment by OmgImAlexis** [comment]
Welcome back @TagForce, good luck reading into the 1k+ commits we've added. :stuck_out_tongue_winking_eye:

**Comment by miigotu** [comment]
and the like 4k in the old repo since he was here last? xD

**Comment by fernandog** [comment]
@TagForce welcome back!

**Comment by neoatomic** [comment]
@TagForce Glad you have you around again.!

**Comment by Thraxis** [comment]
@SickRage/collaborators  @SickRage/moderators 

# Chrome Extension Available
The SickRage Connect Chrome Extension is now out.
It hasn't yet propagated into the Extension Store search, but it is up and available.

[SickRage Connect](https://chrome.google.com/webstore/detail/sickrageconnect/ehknadckoahaedhginedlemfppdnjadm)
[SickRage Connect Github](https://github.com/SickRage/SickRageConnect)
[SickRage Connect Issues](https://github.com/SickRage/SickRageConnect/issues)

**Comment by adaur** [comment]
What about Firefox? ;)

**Comment by miigotu** [comment]
lol who uses FireCrotch anymore? xD

**Comment by labrys** [comment]
@miigotu 19.2% according to http://www.w3schools.com/browsers/browsers_stats.asp

**Comment by ctrlbrk42** [comment]
I got auto switched to the other repo like many/most.  But then it started crashing almost nightly with an out of memory exception (I have plenty of memory).  I started looking into it and realized what happened, so have changed the git source and reset and now hoping the crashes will stop.  Thanks for being professional guys, the other guy clearly is not.

**Comment by simonwicks** [comment]
Hey everyone, 

I'm a Synology user, but I've not updated my version since just before autoProcessTV was removed.

I've not been able to SSH into my NAS in order to put the new post processing files in it, so because everything was still working for me I just left it as it was. 

Now you guys have moved, I'd like to get away from the old version and move to the new one if possible, and stay up to date. 

I've seen the guide for Synology users on switching to the new version (thanks very much for that), but do I still need to manually put the new post processing files in myself in order for that to work?

Currently on SR Version:	BRANCH: (master) / COMMIT: (297ccc3043957ffa96fa142035a9393845cdb024) if that's any help at all.

**Comment by JoLeander899** [comment]
Thank you and stand-up applause to this awesome group!  I also got caught for a few days not knowing what happened.  In answer to simonwicks just as a note -- I needed to put together a new PC at the same time as the change happened to have made the move from windows to linux (and it ROCKS!).  When looking at the nzbtomedia postprocessing area I decided to go straight to the github for it and keep it up-to-date directly myself.  Having it included in contrib on SR is great (don't get me wrong there) but postprocessing is working great for me using this combination on both SR and CP.

**Comment by miigotu** [comment]
@JoLeander899 I would rather not include nzbtomedia in our source tbh, but there are some users who would stab me in the neck with a rusty screwdriver if it were removed from the source =P

**Comment by simonwicks** [comment]
@miigotu Hey, are you saying that the files needed for post processing are included then? I don't need to manually put the files in the /contrib folder myself? From what I'd read before I thought that's what I'd need to do, which is why I held off updating because I've not been able to SSH into the Nas in order to do that myself.

**Comment by miigotu** [comment]
The files are included as a git submodule. The problem is that the Synology package does not initialize the submodule when it installs it, but the first update or branch checkout from inside SR will initialize the submodule. So If you installed the new repo with the Syno package (adjusting the repo url during install), started the package, and then entered your github username/password so you can see branches other than master, and checkout develop branch from inside the interface, it would propagate contrib/nzbtomedia. You could then checkout master again.

**Comment by simonwicks** [comment]
@miigotu Right, thanks for the answer, that makes more sense to me now. I'll give it a go.

**Comment by simonwicks** [comment]
@miigotu Done, hopefully. I imported my backup, and the post processing tab looks the same as it used to be. I thought I'd have to set something up again (part reason I was holding off as well as it sounded a bit confusing), but from this it looks like it's just taken all the old settings. Thanks for the help!

**Comment by gitawego** [comment]
I've switched as well. the old one just becomes unusable with recent updates. 
well done for keeping the projet healthy!

**Comment by miigotu** [comment]
@Hiltronix @gboddin @Grahamvs @loungebob @Langthjem @Halibutt @feelitloveit @ROpdebee @CarlNeuhaus @calvin-- @mvarrieur  @ProjectLane @coderbone @Nico31300 @justinsg

The team you have been getting updates from for the year since echelon left is here on this Org.
https://github.com/orgs/SickRage/people

Just because he can block people from commenting PR's, and take down his issue tracker, and ban the world from the irc channel or forums to prevent anyone from telling you all the truth does not mean you should not have a choice and be informed of where we all went.

If SRTV isnt getting the love and quality you are used to, feel free to try this org out.

**Comment by labrys** [comment]
@hiltronix Let us know if you have any questions about the change.

**Comment by miigotu** [comment]
The bad things he is spreading about us is really getting old. Lies aren't good for anyone.

**Comment by gboddin** [comment]
All right fair enough. Glad to see my PR for bad 403 on all cloudflare exposed sites has been fixed :)

**Comment by miigotu** [comment]
@gboddin yeah I didnt even know you had PR'd that until tonight. Someone pointed it out to me yesterday and I committed it here (Guess it was a 6am commit when I did that, dunno how it lasted 5 months that way lol)

Anyways thanks.

**Comment by gboddin** [comment]
Oh right, nasty one  :) Good it's fixed, started to cause trouble last week for me actually

**Comment by Yellow1144** [comment]
>The bad things he is spreading about us is really getting old. Lies aren't good for anyone.

All you have to do is let the work, your product, speak for itself. It's why I'm here and a bad product is why I abandoned the other one.

**Comment by miigotu** [comment]
@mescribe0 @tresni

Might not want to waste your time with a single dictator who wont accept your code anyways. All other of the team has moved to this Org/Repo.

**Comment by adaur** [comment]
@Strudsen hi, could you consider posting your PR here as well? we accept new code ;)

**Comment by coderbone** [comment]
hey, so I'm giving your fork a try and the activity level looks great so
that's encouraging and I have a juicy little bug fix that I'll make a PR
for later today

main problem though is the broken DB update, I know that it's recommended
that db files be wiped and rebuilt and have gone down that road.... 4hrs in
and it's less than half done :(... if the idea is to get the wider
community to switch to your fork I think you should prioritize getting that
working again.  plus, at some point you'll want to make a schema change and
then you'll need a working update mechanism.  I did have a shot at it but
there were a few issues that someone with more experience on this code base
will no doubt be able to resolve much easier.

**Comment by miigotu** [comment]
@coderbone broken db update?

**Comment by adaur** [comment]
You can always export your DB and import it to your new installation through the UI

**Comment by coderbone** [comment]
I had a sickbeard.db file from the classic SiCKRAGE and was hoping it would
upgrade, it doesn't.  For one thing DBConnection no longer has a close()
method so the attempt fails before it even gets started.  The close is
important because the upgrade shuffles the files around.  The way the
connection is cached (db_cons) further complicates matters as we can't just
re-add the close method.  I did add a close method that also cleaned up
db_cons and also a reopen method to re-attach a valid sqlite3 connection to
the cached connection... that all seemed to work fine but it still
complained about tv_shows .vs. tmp_tv_shows, then another moan about trying
to add 29 values to a 17 column table (or something)... by that point I was
well in over my head and decided to just bite the bullet and do a build
from scratch.  Should be trivial to repro, simply feed SickRage a SiCKRAGE
db file.

Oh, and that reminds me of another issue that must be annoying for first
timers... when doing bulk 'add existing shows' and the searcher only comes
up with one match it may as well just pick it.

**Comment by miigotu** [comment]
@coderbone I think you mean a classic "Sick-Beard" db? Because both SiCKRAGETV and SickRage use the same database version.

**Comment by coderbone** [comment]
good tip... so perhaps the upgrade mechanism should leverage the
export/import code instead of attempting an in-place edit... I'd rate this
as fairly important as it's a "first impressions" kind of thing, people who
switch will just want to point at the new repo, pull and restart, any
little bump or extra few steps will cause some people to go back to where
they felt safe but if the db is thrashed and they end up rebuilding from
scratch on the SiCKRAGE fork it will be you guys that get the heat... just
saying.

**Comment by coderbone** [comment]
I'm talking about the actual 'sickbeard.db' file and the code in db.py
(_processUpgrade) which gets triggered when starting your fork with a file
that was produced by SiCKRAGETV.

In db.py there is a DBConnection class and it doesn't have a close() method
but _processUpgrade attempts to call it anyway just before it calls
restoreDatabase and upgradeDatabase.  The code is broken.

db.py, and it's use of db_cons without consideration of the row_type is
also worrying.... the row factory method of DBConnection should almost
certainly be a static method and db_cons be keyed on both filename and
row_type... if people try to use the webapi (where dict row_type is
requested) it will almost certainly make the other threads that are using
the connection go a little nuts.

**Comment by miigotu** [comment]
A db created with SiCKRAGETV doesnt get upgraded when switching to this fork, we are both on DB version 42. and the sqlite connection instance (which db_cons holds references to) does have a close method. IDK where you are getting these ideas or confusion from.

**Comment by coderbone** [comment]
Yes, the sqlite3 connection has a close method, the DBConnection wrapper
class does not and that's what's being invoked.

Out of curiosity, did you try to reproduce by feeding SickRage a SiCKRAGETV
sickbeard.db file?

Let's not worry about it, I'm passed that thanks to a rebuild from scratch
and am much more interested in getting resource consumption down.

**Comment by miigotu** [comment]
There is no db upgrade coming from SiCKRAGETV to here, they are the same version. And yes my db is from SRTV.

**Comment by coderbone** [comment]
Yep, aware that you are not taking code from SiCKRAGETV.... interesting
that you're started with a db file from the old code, I'll dig deeper and
figure out why it's failing for me.... in the meantime though you should
look at line 486 from your develop branch and note:

                    # close db before attempting restore
                    connection.close()

The 'connection' object here is a DBConnection, which has no close()
method... that close() method was stripped out over a year ago by echel0n
(9466bddc3ee2679fe476b02d94b6227381536637).. ie: before the split...  and
therefore the upgrade mechanism has been broken for at least that long...
on closer review just about everything I hate in db.py happened at that
time.

**Comment by miigotu** [comment]
I agree that taking a db from Sick-Beard does not work/upgrade, but coming from SiCKRAGETV there is no issue.

**Comment by coderbone** [comment]
Figured it out... this pretty much cements my decision to fully switch to
your fork... what has happened is that the SRTV fork has screwed up it's
db.py and as of November 26th has not had a working checkDBVersion().  So
every time I'd started SRTV after Nov 26th I was triggering code that
attempts numerous dodgy updatedb attempts and then finishes up by setting
the db_version to 1!  As I started using your fork with a sickbeard.db file
that was created by damaged code I was doomed before I started.

Which brings me back to a thought from yesterday... users who migrate to
your fork are going to be arriving with damaged sickbeard.db files just
like I did... so perhaps I should put together a PR with code that looks
for this problem and repairs it so people have a smooth transition... sound
reasonable?

**Comment by miigotu** [comment]
Yes, I also have started on a PR that makes db upgrade from Sick-Beard partially work, maybe you could use it to finish that up?
http://pastie.org/private/crs91ejeudq2kpbqwxegq

I started up Sick-Beard and let it create a db, deleted all db from my test environment for SR, and copied the SB databases to SR. Started up SR and repeated several times to get this far.

As that diff sits, it does upgrade the db, but has some exceptions on the first startup (missing a with db_lock[self.filename] somewhere I think, as a thread is accessing the db before the upgrade is complete and `indexer_id` is not a column yet)  These exceptions force you to need to shut down from the interface because the console doesnt accept the ctrl+c event, but on restart there is no more errors and it runs fine (db v42).

**Comment by rr0ss0rr** [comment]
Switched yesterday from sickragetv .. Updated in place with no issues (making sure I had backups of the .db and .ini files) .. Mac OSX 10.11 .. Glad I did .. Thanks

**Comment by coderbone** [comment]
I have a much simpler proposed fix.  The close() logic will eventually have
to be introduced but for now it looks rock solid to simply detect and
correct the problem as a pre-upgrade step (two lines of code plus a
comment).  PR coming soon.

**Comment by miigotu** [comment]
@coderbone my diff I pasted was to fix upgrading from an old Sick-Beard db. Different problem than you were having, but a problem nevertheless.

**Comment by coderbone** [comment]
that's the second time you've talked about migration from sickbeard...
that's not what I've been doing... I guess the confusion comes in because
of the name of the file

**Comment by labrys** [comment]
@ograff The user agents have already been fixed on this repository.
@strudsen DanishBits provider has already been added on this repository
@djang0 If you would like to see your contributions put to use, you should update to this repository as we accept PRs and input from users here

**Comment by labrys** [comment]
@mescribe0 The user agents have already been fixed on this repository.

**Comment by labrys** [comment]
@freedomisforhippies As the old repo isn't really accepting PRs, I'd recommend submitting your PR to the new repo here at https://github.com/SickRage/SickRage.  If you choose to, I have some comments and suggestions for you.

**Comment by miigotu** [comment]
@labrys his PR upgrades database

**Comment by labrys** [comment]
@bencer As that repository is not accepting PRs, consider submitting your PR to this repository instead.

**Comment by OmgImAlexis** [comment]
@SanctiMortem you should be able to get help in this issue and there's a guide on moving over from sickragetv at the top.
Ref: https://github.com/SickRage/sickrage-issues/issues/432

**Comment by SanctiMortem** [comment]
👍🏻 Thank you! I've been confused about everything that was going on!

Just out of curiosity, did my SR Version: Branch supposed to change? 

Current: SR Version: BRANCH: (master)/COMMIT: (9226ba5efba06fb9defc5272f03dae5d6d1c4145)

Apologies if this is a stupid question, I just want to be sure I didn't messed up something :/

Thank you!

**Comment by labrys** [comment]
@SanctiMortem Branch, no, Commit yes... That commit is the old repository.  Easiest way to see if you are on this one is look at either News or the Changelog.  News starts with Happy New Year, changelog has us on version >=5.1

**Comment by SanctiMortem** [comment]
@labrys Thank you, I looked and I'm clearly on the old one. I tried it again and this was my log:

Last login: Thu Jan  7 00:32:27 on console
Gabriels-Mac-mini:Sickrage GaboAsse$ git remote set-url origin https://github.com/SickRage/SickRage.git
Gabriels-Mac-mini:Sickrage GaboAsse$ git fetch origin
error: Unable to append to .git/logs/refs/remotes/origin/master: Permission denied
error: Cannot update the ref 'refs/remotes/origin/master'.
From https://github.com/SickRage/SickRage
 ! 9226ba5...4eccee2 master     -> origin/master  (unable to update local ref)
error: Unable to append to .git/logs/refs/remotes/origin/develop: Permission denied
error: Cannot update the ref 'refs/remotes/origin/develop'.
 ! 9226ba5...a3ce56b develop    -> origin/develop  (unable to update local ref)
 * [new tag]         V4.2.0.01  -> V4.2.0.01
 * [new tag]         v4.2       -> v4.2
 * [new tag]         v4.2.0.1   -> v4.2.0.1
 * [new tag]         v4.2.0.2   -> v4.2.0.2
 * [new tag]         v4.2.0.3   -> v4.2.0.3
 * [new tag]         v4.2.0.4   -> v4.2.0.4
 * [new tag]         v4.2.0.5   -> v4.2.0.5
 * [new tag]         v4.2.0.6   -> v4.2.0.6
 * [new tag]         v4.2.1.0   -> v4.2.1.0
 * [new tag]         v4.2.1.01  -> v4.2.1.01
 * [new tag]         v4.2.1.02  -> v4.2.1.02
 * [new tag]         v4.2.1.03  -> v4.2.1.03
 * [new tag]         v4.2.1.04  -> v4.2.1.04
 * [new tag]         v4.2.1.05  -> v4.2.1.05
 * [new tag]         v4.2.1.06  -> v4.2.1.06
 * [new tag]         v4.2.1.07  -> v4.2.1.07
 * [new tag]         v5.0       -> v5.0
 * [new tag]         v5.0.1     -> v5.0.1
 * [new tag]         v5.0.2     -> v5.0.2
 * [new tag]         v5.0.3     -> v5.0.3
 * [new tag]         v5.1       -> v5.1
Gabriels-Mac-mini:Sickrage GaboAsse$ git reset --hard origin/master
HEAD is now at 9226ba5 HOTFIX: Fixed bug that was causing login screen to display when no user/pass was set in config

It says Permission denied, am I doing something wrong?

**Comment by miigotu** [comment]
Yes, you need to check who owns the `Sickrage GaboAsse/.git` folder and run it as that user. Since you are on mac I am assuming that is admin (root).

**Comment by SanctiMortem** [comment]
I am on that user :s 
so confused haha

**Comment by SanctiMortem** [comment]
Here is my log,

dsenableroot:: ***Successfully enabled root user.
Gabriels-Mac-mini:~ GaboAsse$ cd /Applications/SickRage
Gabriels-Mac-mini:SickRage GaboAsse$ git remote set-url origin https://github.com/SickRage/SickRage.git
Gabriels-Mac-mini:SickRage GaboAsse$ git fetch origin
remote: Counting objects: 5, done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 5 (delta 0), reused 0 (delta 0), pack-reused 1
error: insufficient permission for adding an object to repository database .git/objects
fatal: failed to write object
fatal: unpack-objects failed

**Comment by miigotu** [comment]
seems at some point you used sudo.

back up your db/settings from inside SR, re-clone, and restore the backup.

**Comment by SanctiMortem** [comment]
Ok, 

I re-cloned, I am now on the correct one!

Thank You!

**Comment by OmgImAlexis** [comment]
@miigotu for reference on OSX `/Applications` is typically owned by Root. Since the programs is meant to be run as a daemon it should have been installed to `~/Applications`.

**Comment by SanctiMortem** [comment]
Thank you, all of you for clarifying and helping me thru :) 
👍🏻

**Comment by kooolkat357** [comment]
Hi, Im´m using the old repo on a Readynas ultra and would like to know what i need to do to use this new repo as the old one is starting to get buggy.

**Comment by duramato** [comment]
@kooolkat357 if you have ssh ability you can most likely execute the commands on the first post.
But dont use a ready nas myself

**Comment by OmgImAlexis** [comment]
@joshsalvi

**Comment by miigotu** [comment]
@kooolkat357 I have a package for SR on readyNAS, but it has not been tested yet.

I had tried it on a readyNAS and had to make some adjustments, but have not tested it since.
https://github.com/miigotu/sickrage-readynas

**Comment by labrys** [comment]
@ljw1 @swarwick @AdamBunn @dabadgreek @ChipSkylark37 @scbrown @davidcastellani  

If you want a working, updated SickRage, the [SickRage](https://github.com/SickRage/SickRage) development team moved here.  We have made quite a few changes and are miles ahead of the old repository (nearing 2000 commits).  If you would like any assistance switching to the new repository please let us know or join us on the [SickRage IRC channel](https://kiwiirc.com/client/irc.freenode.net/?theme=basic#sickrage-issues).

**Comment by labrys** [comment]
@Codicier @outkastm @ChipSkylark37

If you want a working, updated SickRage, the [SickRage](https://github.com/SickRage/SickRage) development team moved here.  We have made quite a few changes and are miles ahead of the old repository (nearing 2000 commits).  If you would like any assistance switching to the new repository please let us know or join us on the [SickRage IRC channel](https://kiwiirc.com/client/irc.freenode.net/?theme=basic#sickrage-issues).

**Comment by labrys** [comment]
@L3n-M3n @BaukeFahner @delormev @neclimdul 

If you want a working, updated SickRage, the [SickRage](https://github.com/SickRage/SickRage) development team moved here.  We have made quite a few changes and are miles ahead of the old repository (nearing 2000 commits).  If you would like any assistance switching to the new repository please let us know or join us on the [SickRage IRC channel](https://kiwiirc.com/client/irc.freenode.net/?theme=basic#sickrage-issues).

**Comment by labrys** [comment]
@felixbrucker  @nikagl  @sioban  @damselem  @TATUMTOT  @akyag  @gaetan-petit

If you want a working, updated SickRage, the [SickRage](https://github.com/SickRage/SickRage) development team moved here.  We have made quite a few changes and are miles ahead of the old repository (nearing 2000 commits).  If you would like any assistance switching to the new repository please let us know or join us on the [SickRage IRC channel](https://kiwiirc.com/client/irc.freenode.net/?theme=basic#sickrage-issues).

**Comment by duramato** [comment]
@ChipSkylark37 the people that have been working on sickrage moved on to this repo (kicking echle0n in the process ) , the old one is lacking stabilty and actual working updates, thats why the old one isnt working anymore.

**Comment by neoatomic** [comment]
@duramato I actually find it more surprising that some users still believe that nonsense.
You would expect that at least a light would have gone on by now. :P

**Comment by TATUMTOT** [comment]
Thanks moved over yesturday and OMG. EVERYTHING WORKS AGAIN.

**Comment by kooolkat357** [comment]
@milgotu @labrys Thanks, I did a fresh install with the package from RNXtras.com and altered the config   with the new repo url. SickRage now works better than ever.

**Comment by labrys** [comment]
@kooolkat357 @TATUMTOT great to hear.  Let us know if you have any issues.

**Comment by miigotu** [comment]
@kooolkat357 maybe you could test this sometime? https://github.com/miigotu/sickrage-readynas

**Comment by neclimdul** [comment]
?? you've already accepted a PR from me. what's the tag for?

**Comment by labrys** [comment]
@neclimdul yeah i realised after i clicked submit, sorry about that.

**Comment by walterjnr** [comment]
@kooolkat357 Thanks. I've tried reinstalling a few times and changing the repo with the commandline but it never worked for some reason. I gave it another shot just changing the actual url in the config file and it worked. Thanks

**Comment by Grahamvs** [comment]
I just wanted to say thanks for all the great work. I migrated over just before christmas as I realised something was going horribly wrong with SRTV as nothing seemed to work... When I get some time I'll look at learning some python (I'm a c# programmer by trade) so I can help out with some bugs fixes etc...

**Comment by labrys** [comment]
@Grahamvs Any help is always appreciated.
https://github.com/SickRage/SickRage/blob/master/contributing.md

**Comment by thesatman** [comment]
Made the plunge today after having had a cripled system for a month. Always a shame when things go sour. Thank you guys for continuing this project. Is there already a forum for posting questions? Or do we do that here on github?

**Comment by miigotu** [comment]
Generally on IRC is the place to get help/ask questions (freenode, #sickrage-issues), bugs on github (you can ask on irc first), and feature requests on http://feathub.com/SickRage/SickRage

**Comment by Antioch18** [comment]
BitGrab
BitGrabber
BitNab
BitNabber

**Comment by OmgImAlexis** [comment]
@Antioch18 wrong issue.

**Comment by loungebob** [comment]
Sorry, must have missed this. Where does one go for SR issues since after
the move?



*From:* X O [mailto:notifications@github.com]
*Sent:* Friday, January 22, 2016 9:06 AM
*To:* SickRage/sickrage-issues <sickrage-issues@noreply.github.com>
*Cc:* loungebob <rob@phenomenon.ch>
*Subject:* Re: [sickrage-issues] SickRage (#3)



@Antioch18 <https://github.com/Antioch18> wrong issue.

—
Reply to this email directly or view it on GitHub
<https://github.com/SickRage/sickrage-issues/issues/3#issuecomment-173838863>
.

**Comment by OmgImAlexis** [comment]
@loungebob this is fine, @Antioch18 was commenting about another issue.

**Comment by loungebob** [comment]
Version: Branch: master

Commit: 66fa37c556784fa3b41a06bb3043bd854d165c10



Keep having backlog issues, episodes that shouldn’t be in backlog. 2
different issues, one seems to be the torrent file missing so only the
magnet link can be used (that used to work with SR) and the other one is,
downloaded episodes in the correct format (720p HDTV, but preferred 720p
WEB-DL) showing up as only having downloaded a low-quality version. I need
to manually change them to make them disappear.



Error log for the magnet link/missing torrent file issue:



2016-01-22 09:52:27 WARNING  SEARCHQUEUE-DAILY-SEARCH :: Failed to download
any results [66fa37c]

2016-01-22 09:52:27 INFO     SEARCHQUEUE-DAILY-SEARCH :: Downloading a
result from KickAssTorrents at
https://torcache.net/torrent/4E268AE71525B7956628DBEEC350D12C30C828AB.torrent?title=[kat.cr]dcs.legends.of.tomorrow.s01e01.480p.hdtv.x264.mrs.mkv

2016-01-22 09:52:27 INFO     SEARCHQUEUE-DAILY-SEARCH :: Downloading
DCs.Legends.of.Tomorrow.S01E01.480p.HDTV.X264-mRs.mkv from KickAssTorrents

2016-01-22 09:52:25 WARNING  SEARCHQUEUE-DAILY-SEARCH :: Failed to download
any results [66fa37c]

2016-01-22 09:52:24 INFO     SEARCHQUEUE-DAILY-SEARCH :: Downloading a
result from KickAssTorrents at
https://torcache.net/torrent/F1633EE203B2A5C8937805C4AEFA357AFADD158F.torrent?title=[kat.cr]the.100.s03e01.480p.hdtv.x264.mrs.mkv

2016-01-22 09:52:24 INFO     SEARCHQUEUE-DAILY-SEARCH :: Downloading
The.100.S03E01.480p.HDTV.x264-mRs.mkv from KickAssTorrents

2016-01-22 09:52:24 INFO     SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents]
:: Quality of DCs.Legends.of.Tomorrow.S01E01.480p.HDTV.X264-mRs.mkv is SDTV

2016-01-22 09:52:24 INFO     SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents]
:: Quality of The.100.S03E01.480p.HDTV.x264-mRs.mkv is SDTV

2016-01-22 09:52:24 INFO     SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents]
:: Found result The.100.S03E01.480p.HDTV.x264-mRs.mkv at
https://torcache.net/torrent/F1633EE203B2A5C8937805C4AEFA357AFADD158F.torrent?title=[kat.cr]the.100.s03e01.480p.hdtv.x264.mrs.mkv

2016-01-22 09:52:24 INFO     SEARCHQUEUE-DAILY-SEARCH :: [KickAssTorrents]
:: Found result DCs.Legends.of.Tomorrow.S01E01.480p.HDTV.X264-mRs.mkv at
https://torcache.net/torrent/4E268AE71525B7956628DBEEC350D12C30C828AB.torrent?title=[kat.cr]dcs.legends.of.tomorrow.s01e01.480p.hdtv.x264.mrs.mkv

2016-01-22 09:52:22 INFO     SEARCHQUEUE-DAILY-SEARCH :: Beginning daily
search for new episodes

2016-01-22 09:52:20 INFO     DAILYSEARCHER :: No new released episodes
found ...

2016-01-22 09:52:20 INFO     DAILYSEARCHER :: Searching for new released
episodes ...

2016-01-22 09:47:47 WARNING  SEARCHQUEUE-BACKLOG-268592 ::
[KickAssTorrents] :: Failed to download any results [66fa37c]

2016-01-22 09:47:47 INFO     SEARCHQUEUE-BACKLOG-268592 ::
[KickAssTorrents] :: Downloading a result from KickAssTorrents at
https://torcache.net/torrent/F1633EE203B2A5C8937805C4AEFA357AFADD158F.torrent?title=[kat.cr]the.100.s03e01.480p.hdtv.x264.mrs.mkv

2016-01-22 09:47:47 INFO     SEARCHQUEUE-BACKLOG-268592 ::
[KickAssTorrents] :: Downloading The.100.S03E01.480p.HDTV.x264-mRs.mkv from
KickAssTorrents

2016-01-22 09:47:47 INFO     SEARCHQUEUE-BACKLOG-268592 ::
[KickAssTorrents] :: Quality of The.100.S03E01.480p.HDTV.x264-mRs.mkv is
SDTV

2016-01-22 09:47:46 INFO     SEARCHQUEUE-BACKLOG-268592 ::
[KickAssTorrents] :: Found result The.100.S03E01.480p.HDTV.x264-mRs.mkv at
https://torcache.net/torrent/F1633EE203B2A5C8937805C4AEFA357AFADD158F.torrent?title=[kat.cr]the.100.s03e01.480p.hdtv.x264.mrs.mkv

2016-01-22 09:47:46 INFO     SEARCHQUEUE-BACKLOG-268592 ::
[KickAssTorrents] :: Performing episode search for The 100

2016-01-22 09:47:43 INFO     SEARCHQUEUE-BACKLOG-268592 :: Beginning
backlog search for: [The 100]





*From:* X O [mailto:notifications@github.com]
*Sent:* Friday, January 22, 2016 10:03 AM
*To:* SickRage/sickrage-issues <sickrage-issues@noreply.github.com>
*Cc:* loungebob <rob@phenomenon.ch>
*Subject:* Re: [sickrage-issues] SickRage (#3)



@loungebob <https://github.com/loungebob> this is fine, @Antioch18
<https://github.com/Antioch18> was commenting about another issue.

—
Reply to this email directly or view it on GitHub
<https://github.com/SickRage/sickrage-issues/issues/3#issuecomment-173853982>
.

**Comment by miigotu** [comment]
@loungebob you would create a new issue in this repo, not use this issue itself for bug reports.

**Comment by thesatman** [comment]
can we still use [Sick Beard Index] at lolo.sickbeard.com ? I am getting Connection error to getURL.Or is this only to be used with SickRageTV?

**Comment by miigotu** [comment]
@thesatman no-one except sickbeard itself is supposed to use SBI.

However we do have it working, but the site is down:

```
Error 503 Service Unavailable

Service Unavailable

Guru Meditation:

XID: 1835397661

```

**Comment by loungebob** [comment]
https://github.com/SickRage/SickRage there? Cant see an Issues part…



*From:* miigotu [mailto:notifications@github.com]
*Sent:* Friday, January 22, 2016 10:47 AM
*To:* SickRage/sickrage-issues <sickrage-issues@noreply.github.com>
*Cc:* loungebob <rob@phenomenon.ch>
*Subject:* Re: [sickrage-issues] SickRage (#3)



@loungebob <https://github.com/loungebob> you would create a new issue in
this repo, not use this issue itself for bug reports.

—
Reply to this email directly or view it on GitHub
<https://github.com/SickRage/sickrage-issues/issues/3#issuecomment-173863507>
.

**Comment by miigotu** [comment]
@loungebob https://github.com/SickRage/sickrage-issues/issues

**Comment by labrys** [comment]
@cgoconseils We are already using https for torrentleech in this repository.

If you want a working, updated SickRage, the [SickRage](https://github.com/SickRage/SickRage) development team moved here.  We have made quite a few changes and are miles ahead of the old repository (nearing 2000 commits).  If you would like to switch over and need any assistance switching to the new repository please let us know or join us on the [SickRage IRC channel](https://kiwiirc.com/client/irc.freenode.net/?theme=basic#sickrage-issues).

**Comment by wigwam12** [comment]
Could someone guide me.
New to synology and started using Sickrage(TV) and loved it.
Installed via the Synco packages
It broke after the recent update and won't start so I can't make a backup. I THINK I have made one previously but not sure (where do I look?)
Anyway - I have just tried Sonarr and looks nice but I do miss the familiarity of Sickrage v having to learn yet another app.

So:
1) How do I switch to the new repo please?
2) will I be able to retrieve all my settings, indexers, shows from old Sickrage repo?

**Comment by neoatomic** [comment]
@wigwam12 Its very easy, see : https://github.com/SickRage/sickrage-issues/wiki/Switching-your-Synology's-Sickrage-to-the-new-repository

And you have all your shows & settings back.
(You might need to rename sickrage.db to sickbeard.db depending on what version you previously had.)

**Comment by wigwam12** [comment]
Thanks.
That seems pretty straighforward (famous last words!)
The version I have now is the Sickrage Repo managed by echelon that was installed from the syncocommunity packages as "sickrage" - Master branch.

The other things is that I have just navigated to /usr/local/sickrage/var/backup and seen that I have 2 backup zip files - ones that is 11mb from Dec and a 5mb one from 2 weeks ago.
Will the newer (smaller one) be corrupted? Nervous about the discrepancy in size.
Where do I upload the zip files back for the new Repo to see them?

**Comment by wigwam12** [comment]
Neoatomic I am racking my brains to work out where our paths have crossed before as I know you have kindly helped with some stuff in the past!

**Comment by neoatomic** [comment]
@wigwam12 , Well in case you hadn't heard, the whole development team (including me) moved here. 
So probbebly i helped you out when we where still on the old sickragetv repo. :)

I would use the older backup from December, just move the backup file to the `video` folder. (or another easy to find Syno folder) And then restore it from within Sickrage, that should be all.

**Comment by wigwam12** [comment]
so far so good!
The folder being sickbeard-custom rather than sickrage will take a little getting used to I suspect but hopefully I won't need to access it at all

How do I create a folder called "backup" in sickbeard-custom to store the backups in? just like things to be tidy. I manually created one use using win scf but it says backup failed when I try to backup to it

**Comment by labrys** [comment]
@wigwam12 please keep this issue on topic.  Minor questions can be answered in our IRC channel or you can create a new issue for more import issues.

**Comment by wigwam12** [comment]
Apologies - got carried away with excitement at it working again

**Comment by walterjnr** [comment]
Is there any chance someone could create a working package for Readynas OS6? I was using the version from RNXtras.com, which uses the old repo, and it seems like he has abandoned his site. After typing some of the earlier commands to switch repos it updated fine but now a new update has killed it again.

**Comment by miigotu** [comment]
@walterjnr This is the wrong place to ask that, but there are 2 working packages for RN OS6.
https://github.com/miigotu/sickrage-readynas
https://github.com/mhynlo/rn-sickrage

**Comment by walterjnr** [comment]
Thanks. I missed that one. Much appreciated.

**Comment by miigotu** [comment]
Next time, if you see someone selling open source packages you should demand they remove them and never pay them a dime.

**Comment by OmgImAlexis** [comment]
We don't really need this anymore so closing.
